### DISCUSSION TITLE
Is is possible to skip "problematic" files during a new scan?

### DISCUSSION THREAD

**Discussion by justinosdotnet** [photoview/photoview#636] [Q&A and Troubleshooting]
I am receiving errors on certain (very, very few) files during the initial scan on new install.

Would it be possible to flag and skip over said erroneous files during the next scans, as to avoid the repetitive scanner error messages every time I scan for new files?  This gets a bit annoying since I have it set up to periodically rescan every hour.  The error messages pile up on each other.  (Granted, the logical move would be to eliminate the problematic files at the source, but unfortunately my system is setup in a drive syncing fashion so any files I would delete would simply reappear to be scanned the following hour)

Right now I only have 2 files failing out of a couple thousand, so it's definitely not a major problem.  But I was still curious to know if it might be possible to tag these files as errors and skip them (or at least the error messages) from future scans.

Just as reference, here are the 2 errors:

Scanner error
Failed to begin database transaction: failed to process photo (/photos/IMG_1782.MOV): could not encode mp4 video (/photos/IMG_1782.MOV): encoding video using: /usr/bin/ffmpeg: exit status 1

Scanner error
Failed to begin database transaction: failed to process photo (/photos/Melanie photo/IMG_5464.HEIC): creating high-res cached image: image decoding: failed to decode HEIF image (/photos/Melanie photo/IMG_5464.HEIC): Invalid input: Unexpected end of file: Extent in iloc box references data outside of file bounds (points to file position 151977)

**Comment by viktorstrate** [comment]
There is no good way to do it currently. Your best option at the moment would be to add the files to a `.photoviewignore` file.
The best solution would probably be to somehow mark unprocessable files in the database and skip them in future scans.

**Reply by justinosdotnet** [reply]
EDIT/UPDATE: Never mind, I found a bit more information around here and got my answer.  There is not much info on photoviewignore on the website, but it does appear to fix my problem and first glance, and accomplish what I was hoping to do.  I will simply keep updating the file with further problematic file names should the situation arise again.  Thank you very much!

--------------------------------

Thanks for this.  I have sinced looked up how to use the photoviewignore file and can't find the details as to how to integrate it correctly.  If I understand correctly i would simply add the file names to a .photoviewignore file, but I'm uncertain where to place the file.  I read in the root directory, but i'm unsure what root dir we are referring to.  In the actual photo dir itself, or the folder where I launch the program (ie yml file), or withing in container?  If the later, I'm not exactly experienced enough with docker to figure out how to do this.
